server/src/repositories/trees.repository.ts

```diff
@@ -13,4 +14,18 @@ export class TreesRepository extends DefaultCrudRepository<
   ) {
     super(Trees, dataSource);
   }
+
+  async getEntityIdsByOrganizationId(organizationId : Number):Promise<Array<Number>>{
+    expect(organizationId).number();
+    expect(this).property("execute").defined();
+    const result = await this.execute(`select * from getEntityRelationshipChildren(${organizationId})`, []);
+    return result.map(e => e.entity_id);
+  }
+
+  async getPlanterIdsByOrganizationId(organizationId : Number):Promise<Array<Number>>{
+    expect(organizationId).number();
+    expect(this).property("execute").defined();
+    const result = await this.execute(`select * from planter where person_id in (select entity_id from getEntityRelationshipChildren(${organizationId}))`, []);
```

Comment:
If the intention is to get all planter ids within an organization, then this query needs to look up planters by organization_id matching the entities in getEntityRelationshipChildren

Reply:
So I need to change 'person_id' to 'organization_id' that's all right?

Reply:
@deepwinter